paper-drawer-panel.html

```diff
@@ -757,9 +766,43 @@
 
         _moveDrawer: function(translateX) {
           this.transform(this._transformForTranslateX(translateX), this.$.drawer);
-        }
+        },
+
+        _getAutoFocusedNode: function() {
+          return Polymer.dom(this).querySelector('[autofocus]') || this.$.firstTabStop;
```

Comment:
only content in the light dom. If the final [autofocus] element is behind some shadow root, then the element's author should delegate focus when it receives focus.